<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MInstoreProcess</class>
 <widget class="QDialog" name="MInstoreProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>771</width>
     <height>381</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>快递单号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>快递公司</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>收件地址</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>生成取件码</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>生成入库单号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>操作</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>入库操作</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>操作驿站ID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>驿站名称</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_sname"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>驿站地址</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="textEdit_spos"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_sid"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>操作时间</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>操作人员ID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_wid"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>新增入库</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>610</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除入库</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_commit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>610</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>提交入库</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>610</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消操作</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
